### ISSUE TITLE
How to use result of LSTM after fitting the model

### ISSUE THREAD

**Issue by phdrsch** [keras-team/keras#14381]
This is how I built and compiled my LSTM model

model = Sequential()
model.add(LSTM(40, input_shape=(24, 12), return_sequences=False))
model.add(Activation('relu'))
model.add(Dense(units=1))

from keras import optimizers
adam = optimizers.Adam(lr=0.001, beta_1=0.9, beta_2=0.999, epsilon=1e-10, decay=0.0, amsgrad=False)
model.compile(loss='mse', optimizer=adam, metrics=['mae'])
callbacks = [EarlyStopping(monitor='val_loss', patience=20), ModelCheckpoint(filepath='LSTM.h5', monitor='val_loss', save_best_only=True)]

Now when I executed it via 

history=model.fit(X_train, Y_train, batch_size=15, epochs=50, validation_split=0.1, verbose=2,callbacks=callbacks) 

where X_train was initially shaped (10000, 12) which I re-shaped to (10000, 24, 12) so that model.fit could work without error. Y_train was shaped (10000, 12). No problem till now.

The problem started when I am using below

estimated_result = model.predict(X_train) - (is prediction like this wrong?)

errors = sum(abs(estimated_result-Target_output))  The estimated_result shape is (10000, 1) and Target_output shape (10000, 12))

final errors = sum(errors != 0)
print(final errors)   
Final errors are coming to be the same no matter whatever input parameters to the code is. I suspect the way I used model.predict is wrong. How do I use the result of LSTM for the error formula?

Also how to correctly re-shape X_train to (10000, 24, 12)? I suspect if the model.predict is not wrong the reshaping might be!! Please advise what is incorrectly done. Thanks in advance

**Comment by phdrsch** [comment]
Can someone pleasee advise?

**Comment by SamMohel** [comment]
did you solve it ?
